web/src/components/InfoBanner.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <v-banner
     v-if="bannerText"
-    color="grey-darken-1"
+    bg-color="grey-lighten-6"
+    sticky
+    :style="{ top: '64px' }"
```

Comment:
Is this inline style needed? The banner stays the same when I remove it locally.
```suggestion
```

Reply:
does it not scroll behind the app bar for you?

Reply:
oh, i didn't try scrolling - good catch :slightly_smiling_face: